services/QuillLMS/spec/models/classroom_unit_activity_state_spec.rb

```diff
@@ -97,19 +97,19 @@
       create(:classrooms_teacher, user_id: teacher.id, classroom_id: classroom2.id)
       cua2.save
       lesson_data = { 'classroom_unit_id' => cua.classroom_unit.id,
```

Comment:
If you move the first key-value down a line, this will get rid of the added whitespace:

```
lesson_data = { 
  'classroom_unit_id' => cua.classroom_unit.id,
  'classroom_unit_activity_state_id' => cua.id,
  'activity_id' => cua.unit_activity.activity.id,
  'activity_name' => cua.unit_activity.activity.name,
  'unit_id' => cua.classroom_unit.unit_id,
  'completed' => cua.completed,
  'visible' => unit_activity.visible 
}
```